test/BinaryParse.cpp

```diff
@@ -435,15 +341,14 @@ TEST_F(BinaryParseTest, ExtendedInstruction) {
   };
   const auto instruction = MakeInstruction(
       SpvOpExtInst, {2, 3, 1, uint32_t(OpenCLLIB::Entrypoints::Sqrt), 4});
-  const spv_parsed_instruction_t parsed_inst = {instruction.data(),
-                                                uint16_t(instruction.size()),
-                                                SpvOpExtInst,
-                                                SPV_EXT_INST_TYPE_OPENCL_STD,
-                                                2,  // type id
-                                                3,  // result id
-                                                operands.data(),
-                                                uint16_t(operands.size())};
-  EXPECT_THAT(instructions(), ElementsAre(_, ParsedInstruction(parsed_inst)));
+  EXPECT_CALL(client_, Instruction(ParsedInstruction(spv_parsed_instruction_t{
+                           instruction.data(), uint16_t(instruction.size()),
```

Comment:
? I don't understand.
That's a constructor cast, which is in C++ but not in C.
(And Andrew found out last week the Google style guide prefers static_cast<uint16_t>(...) here, instead of the constructor cast.  But we have not been doing that very much in this codebase so far.)